client/src/components/settings/toggle-setting.css

```diff
@@ -80,50 +84,55 @@
 
 .toggle-radio-group {
   display: flex;
-  align-items: start;
 }
 
 .toggle-radio-group label {
-  font-weight: normal;
-  display: flex;
-  align-items: center;
+  margin: 8px;
```

Comment:
I don't think you want to put a left margin on the labels. If you narrow the browser in far enough so the radio buttons drop below the label then the left margin pushes them in that extra `8px`.  Always make sure to test your styles both in narrow and wide view ports and also zoom in up to 400% in a wide view port.

Also, adding a top/bottom margin on these `label`s is causing the radio buttons to not line up vertically with their text to the right. For example, the Private/Public text for the buttons should be lined up vertically with the heading text "My Profile".

Reply:
Ok, ill remove left margin to labels, could you mention the dimensions on which I could check for future and current PR?